### DISCUSSION TITLE
CLI SUPABASE IN VISUAL STUDIO: Unable to paste (or manually type) the access token from https://app.supabase.com/account/tokens

### DISCUSSION THREAD

**Discussion by camaerart** [supabase/supabase#14345] [Questions]
Hi guys, Im following this tuto for creating a custom chat with data using SB,  I got stuck in the CLI part : https://www.youtube.com/watch?v=B6Vxr-ijjrs&t=452s 
The thing is that when I try to paste the token after executing these commands   **npm install supabase --save-dev ,  => npx supabase init => npx supabase login** the terminal does not allow me to paste at least type the token... I already try to run vs as adm and also change the terminal from  power shell to node but nothing works, I don't know exactly how to search for help with this issue, here is a video for context: https://i.imgur.com/qbBN87S.mp4  

thanks so much

**Comment by Daniel-Ash** [comment]
I had this same issue. 
As a workaround I found if add the credential to your initial command:
`npx supabase login {credential}`
Then you can simply hit enter when prompted and it will accept the credential.

Ideally this can get fixed though..

**Reply by camaerart** [reply]
Thanks for your answer, I do replace the credential for my token like this  `npx supabase login {sbp_dc4.......aa88}` ( also with just the token and the common sense variations but nothing works)  and I get this error: Error: unknown shorthand flag: 'e' in -encodedCommand
Usage:
  supabase login [flags]

Flags:
  -h, --help   help for login

Global Flags:
      --debug                             output debug logs to stderr
      --dns-resolver [ native | https ]   lookup domain names using the specified resolver (default native)
      --experimental                      enable experimental features
      --workdir string                    path to a Supabase project directory

**Comment by smcady** [comment]
Same issue here.  Cannot login to my project from terminal at all.  I am a complete newb with supabase, and I am sure I am just missing something simple. In configuring my project locally in vs code, when I attempt to login with "supabase login", I am prompted to provide an access token. I have retrieved the token from my project, but I am simply unable to enter it in terminal. Typing appears to be disabled when the prompt is presented, and pasting also appears not to work. No matter what I do, I cannot enter this token. Here is a screencap of my terminal window after trying unsuccessfully to enter the token. Any ideas what I am missing?
![image](https://github.com/supabase/supabase/assets/2520946/6a315fb1-8e90-46f8-8357-eac72912edf3)

**Reply by smcady** [reply]
Thanks everyone. In the end, I was finally able to paste the key with the unlikely keystroke combo SHIFT + INSERT 
First time for everything I guess.

**Reply by codeconvert** [reply]
hey. still struggling with this. I'm finding it infuriating now! I wish one of their tutorial videos covered this point for newbies. I've tried doing shift + insert but no luck. I'm on windows, are you on mac?

**Reply by Daniel-Ash** [reply]
I recently encountered this again when setting up a new machine. My experience (on windows) was that shift + insert did not display any text being inserted in the terminal, but still worked after hitting enter..

**Comment by krystynak** [comment]
On MacOS
pbpaste | supabase login 
may work
